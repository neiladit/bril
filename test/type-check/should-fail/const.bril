main{

    a: int =  const true;
    print a;
}
