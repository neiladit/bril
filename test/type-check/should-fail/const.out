line 1: assigning bool value to int variable
const error!
